src/apexpy/apex.py

```diff
@@ -76,15 +74,15 @@ def __init__(self, date=None, refh=0, datafile=None, fortranlib=None):
             fortranlib = fa.__file__
 
         self.RE = 6371.009  # mean Earth radius
-        self.set_refh(refh) # reference height
+        self.set_refh(refh)  # reference height
 
         if date is None:
             self.year = helpers.toYearFraction(dt.datetime.now())
         else:
             try:
                 # convert date/datetime object to decimal year
                 self.year = helpers.toYearFraction(date)
-            except:
+            except AttributeError:
                 # failed so date is probably int/float, use directly
```

Comment:
```suggestion
                # Failed while finding datetime attribute, so
                # date is probably an int or float; use directly
```

Just clarifying why the AttributeError is raised.